docs/config_entries_config_flow_handler.md

```diff
@@ -258,6 +258,8 @@ class OAuth2FlowHandler(
 
     async def async_step_reauth(self, user_input=None):
         """Perform reauth upon an API authentication error."""
+        # OPTIONAL: Cache the original input for use in description placeholders.
```

Comment:
Maybe make another example with this approach? The oauth2 flow normally doesn't use username in the config entry, so it's confusing to include that here.